js/ui/toolbar/ui.toolbar.base.js

```diff
@@ -149,12 +149,38 @@ var ToolbarBase = AsyncCollectionWidget.inherit({
         this.setAria("role", "toolbar");
     },
 
+    _waitParentAnimationFinished: function() {
+        var $element = this.$element(),
+            timeout = 15;
+        return new Promise(function(resolve) {
+            const check = function() {
+                var animated = true;
+                $element.parents().each(function(_, parent) {
+                    if($(parent).data("dxAnimData")) {
```

Comment:
If somebody rename 'dxAnimData' constant the trouble will happen. I suggest using https://js.devexpress.com/Documentation/ApiReference/Common/utils/fx/Methods/#isAnimatingelement method from 'fx' that do completely the same thing.